packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard.tsx

```diff
@@ -0,0 +1,230 @@
+                return (
+                    <span>
+                        {t.acquired_a_NFT_from()} <ReversedAddress address={feed.metadata?.from} />
+                    </span>
+                )
+            }
+        }
+        if (feed.tags?.includes('Token') || feed.tags?.includes('ETH')) {
+            if (feed.metadata?.from?.toLowerCase() === address) {
+                return (
+                    <span>
+                        {t.sent_to()} <ReversedAddress address={feed.metadata?.to} />
+                    </span>
+                )
+            }
+            if (feed.metadata?.to?.toLowerCase() === address) {
+                return (
+                    <span>
+                        {t.received_from()} <ReversedAddress address={feed.metadata?.from} />
+                    </span>
+                )
+            }
+        }
+        if (feed.tags?.includes('Gitcoin')) {
+            if (feed.metadata?.from?.toLowerCase() === address) {
+                return t.donated()
+            }
+            if (feed.metadata?.to?.toLowerCase() === address) {
+                return t.received_donation_from()
+            }
+        }
+        if (feed.metadata?.from?.toLowerCase() === address) {
+            return t.received()
+        }
+        return t.sent()
+    }, [address, feed])
+
+    const logo = useMemo(() => {
+        if (feed.tags?.includes('NFT')) {
+            return (
+                <Card className={classes.img}>
+                    <NFTCardStyledAssetPlayer
+                        contractAddress={feed.metadata?.collection_address}
+                        chainId={ChainID[feed.metadata?.network ?? 'ethereum']}
+                        url={resolveIPFSLinkFromURL(
+                            NFTMetadata?.metadata?.imageURL ||
+                                feed.attachments?.find((attachment) => attachment?.type === 'preview')?.address ||
+                                '',
+                        )}
+                        tokenId={feed.metadata?.token_id}
+                        classes={{
+                            loadingFailImage: classes.loadingFailImage,
+                            wrapper: classes.img,
+                            iframe: classes.img,
+                        }}
+                    />
+                </Card>
+            )
+        }
+        if (feed.tags.includes('Token') || feed.tags.includes('ETH')) {
+            return (
+                <TokenIcon
+                    pluginID={NetworkPluginID.PLUGIN_EVM}
+                    chainId={ChainID[feed.metadata?.network ?? 'ethereum']}
+                    address={feed.metadata?.token_address ?? ZERO_ADDRESS}
+                />
+            )
+        }
+        if (feed.tags.includes('Gitcoin')) {
+            return (
+                <img
+                    className={classes.img}
+                    src={feed.attachments?.find((attachment) => attachment?.type === 'logo')?.address}
```

Comment:
The enum name is just for demonstrating purpose.

```suggestion
                    src={feed.attachments?.find((attachment) => attachment?.type === AttachmentType.Logo)?.address}
```

Reply:
the API don't provider such enum, `type` is a property of an object array

Reply:
Does it have a fixed range of valid values of `attachment.type`? Or just any plain string is available.